### DISCUSSION TITLE
Pieces CLI delete chats

### DISCUSSION THREAD

**Discussion by buzby08** [pieces-app/support#453] [Q&A] [answered]
In the pieces CLI, is it possible to force the conversation to be deleted, without the extra confirmation message. For instance, currently, to delete a conversation, you have to run `pieces conversation -d i`, with ***i*** being the index of the chat you wish to delete. Then pieces will ask you to confirm you want to delete the chat. I was wondering if there is a way to ignore this, for example a `--force` flag, resulting in pieces CLI not asking for confirmation.

**Comment by buzby08** [comment]
> In the pieces CLI, is it possible to force the conversation to be deleted, without the extra confirmation message. For instance, currently, to delete a conversation, you have to run `pieces conversation -d i`, with _**i**_ being the index of the chat you wish to delete. Then pieces will ask you to confirm you want to delete the chat. I was wondering if there is a way to ignore this, for example a `--force` flag, resulting in pieces CLI not asking for confirmation.

![image](https://github.com/user-attachments/assets/e15182af-aa1c-439b-8c47-387a67418074)

Here is the image version of my previous message

**Reply by bishoy-at-pieces** [reply, marked as answer]
Great, I created an issue for it here https://github.com/pieces-app/cli-agent/issues/218 feel free to take it up
Also if you have any other suggestion please feel free to create an issue on the Pieces CLI GitHub repository

**Comment by rosie-at-pieces** [comment]
@Bishoy-at-pieces some feedback here!
